client/extensions/wp-job-manager/components/job-listings/index.jsx

```diff
@@ -1,5 +1,275 @@
-const JobListings = () => {
-	return null;
-};
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
 
-export default JobListings;
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormToggle from 'components/forms/form-toggle/compact';
+import SectionHeader from 'components/section-header';
+
+class JobListings extends Component {
+	state = {
+		job_manager_category_filter_type: 'any',
+		job_manager_date_format: 'relative',
+		job_manager_enable_categories: false,
+		job_manager_enable_default_category_multiselect: false,
+		job_manager_enable_types: true,
+		job_manager_google_maps_api_key: '',
+		job_manager_hide_expired: true,
+		job_manager_hide_expired_content: true,
+		job_manager_hide_filled_positions: false,
+		job_manager_multi_job_type: false,
+		job_manager_per_page: 10,
```

Comment:
These are eventually going to come directly from the API, and that's how they are named there. We could normalize them I suppose, but I'm not convinced that it's necessary.